### DISCUSSION TITLE
Allows people to customize which modules are imported when createPinia

### DISCUSSION THREAD

**Discussion by WormGirl** [vuejs/pinia#1634] [Help and Questions]
### What problem is this solving

Sometime,  we need to share some module code across projects，It is unreasonable to load all modules under the stores  directory.

### Proposed solution

// something like this, not load all file at stores
```js
import { createPinia } from 'pinia';
import a from './a';
import b from './b';

export default createPinia({
a,
b
});
```

**Comment by posva** [comment]
They only load when you use them. You can use any of them to explicitly load one

**Reply by WormGirl** [reply]
ok, it is my mistake.I understand the problem.
